sled-agent/src/services.rs

```diff
@@ -1673,10 +1677,19 @@ impl ServiceManager {
 
                 let dns_service = Self::dns_install(info, None, None).await?;
 
+                let clickhouse_server_config =
+                    PropertyGroupBuilder::new("config")
+                        .add_property(
+                            "config_path",
+                            "astring",
+                            format!("{CLICKHOUSE_SERVER_CONFIG_DIR}/{CLICKHOUSE_SERVER_CONFIG_FILE}"),
+                        );
```

Comment:
Also part of the refactoring. Let's use the constants we are using for the configuration files in the SMF service as well, so we don't have to hardcode things into an SMF method script.